STM32/AC/Core/Src/ACBoard.h

```diff
@@ -8,7 +8,28 @@
 #ifndef SRC_ACBOARD_H_
 #define SRC_ACBOARD_H_
 
-void handleUserInputs(const char* startMsg);
+/***************************************************************************************************
+** DEFINES
+***************************************************************************************************/
+
+/* AC board status register definitions */
+
+/* Each of the 5 ports (1x Fan + 4x AC) can be on or off */
+#define AC_BOARD_PORT_x_STATUS_Msk(x) (1U << (x))
+#define AC_BOARD_PORTS_STATUS_Msk     ((1U << 5U) - 1U)
```

Comment:
Is this Msk used?

Reply:
Nope, but it feels right to me. I'll delete if you care strongly.

Reply:
Hm, I get what you are saying and I agree it doesn't really have any major impact. However, if someone actually needed to use this, they would have to update the code anyways, so generally speaking I would leave things out that may or may not be relevant in the future until they become relevant. 

In this instance I don't feel strongly about it though.